content/en/docs/concepts/security/_index.md

```diff
@@ -3,4 +3,245 @@ title: "Security"
+   for applications, based on the level of trust you have in them.
+1. Run a specialized operating system on your nodes that is designed specifically
+   for running containerized workloads. This is typically based on a read-only
+   operating system (_immutable image_) that provides only the services
+   essential for running containers.
+
+   Container-specific operating systems help to isolate system components and
+   present a reduced attack surface in case of a container escape.
+1. Define [ResourceQuotas](/docs/concepts/policy/resource-quotas/) to
+   fairly allocate shared resources, and use
+   mechanisms such as [LimitRanges](/docs/concepts/policy/limit-range/)
+   to ensure that Pods specify their resource requirements.
+1. Partition workloads across different nodes.
+   Use [node isolation](/docs/concepts/scheduling-eviction/assign-pod-node/#node-isolation-restriction)
+   mechanisms, either from Kubernetes itself or from the ecosystem, to ensure that
+   Pods with different trust contexts are run on separate sets of nodes.
+1. Use a {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+   that provides security restrictions.
+1. On Linux nodes, use a Linux security module such as [AppArmor](/docs/tutorials/security/apparmor/) (beta)
+   or [seccomp](/docs/tutorials/security/seccomp/).
+
+### Runtime protection: storage {#protection-runtime-storage}
+
+To protect storage for your cluster and the applications that run there, you can:
+
+1. Integrate your cluster with an external storage plugin that provides encryption at
+   rest for volumes.
+1. Enable [encryption at rest](/docs/tasks/administer-cluster/encrypt-data/) for
+   API objects.
+1. Protect data durability using backups. Verify that you can restore these, whenever you need to.
+1. Authenticate connections between cluster nodes and any network storage they rely
+   upon.
+1. Implement data encryption within your own application.
+
+For encryption keys, generating these within specialized hardware provides
+the best protection against disclosure risks. A _hardware security module_
+can let you perform cryptographic operations without allowing the security
+key to be copied elsewhere.
+
+### Networking and security
+
+You should also consider network security measures, such as
+[NetworkPolicy](/docs/concepts/services-networking/network-policies/) or a
+[service mesh](https://glossary.cncf.io/service-mesh/).
+Some network plugins for Kubernetes provide encryption for your
+cluster network, using technologies such as a virtual
+private network (VPN) overlay.
+By design, Kubernetes lets you use your own networking plugin for your
+cluster (if you use managed Kubernetes, the person or organization
+managing your cluster may have chosen a network plugin for you).
+
+The network plugin you choose and the way you integrate it can have a
+strong impact on the security of information in transit.
+
+### Observbility and runtime security
```

Comment:
Can you fix the spelling error in the heading?

Reply:
Sure!